pkg/api/vminfo.go

```diff
@@ -353,6 +353,13 @@ type ScalingConfig struct {
 	// default.
 	EnableLFCMetrics *bool `json:"enableLFCMetrics,omitempty"`
 
+	// UseSimpleLFCScaling, if true, calculates goal LFC size only based on the largest available
+	// working set size window, instead of trying to allow downscaling at earlier opportunities.
+	//
+	// This is not fit for general use. It's meant as a temproary escape hatch to let us assess the
+	// upper bound of potential improvements to our LFC goal size heuristics.
+	UseSimpleLFCScaling *bool `json:"useSimpleLFCScaling,omitempty"`
```

Comment:
```suggestion
	LFCUseLargestWindow *bool `json:"lfcUseLargestWindow,omitempty"`
```

IMO explains what it does right in the name.

Reply:
Fair enough, yeah -- renamed in 19b5c47603b6cb883026ed526c02b5daee0c0227